packages/web/src/server/index.ts

```diff
@@ -2,7 +2,7 @@
 /* eslint-disable no-console */
 import 'server-only';
 import type { AllowedPropertyValues } from '../types';
-import { isDevelopment, isProduction, parseProperties } from '../utils';
+import { isProduction, parseProperties } from '../utils';
```

Comment:
**praise:** Thanks for the housekeeping. Don't we have eslint configured to catch unused variables?